src/state/trajectory/logics.ts

```diff
@@ -332,10 +332,35 @@ const loadFileViaUrl = createLogic({
     type: LOAD_FILE_VIA_URL,
 });
 
```

Comment:
This logic currently handles the time parameter, but has been written to anticipate handling future URL parameters.